sympy/integrals/tests/test_intpoly.py

```diff
@@ -148,6 +148,21 @@ def test_polytope_integrate():
     assert result_dict[expr2] == S(13062161)/27
     assert result_dict[expr3] == S(1946257153)/924
 
+    #  Tests when all integral of all monomials up to a max_degree is to be
+    #  calculated.
+    assert polytope_integrate(Polygon(Point(0, 0), Point(0, 1),
+                                      Point(1, 1), Point(1, 0)),
+                              max_degree=4) == {0: 0, 1: 1, x: 1 / 2,
+                                                x ** 2 * y ** 2: 1 / 9,
+                                                x ** 4: 1 / 5, y ** 4: 1 / 5,
+                                                y: 1 / 2, x * y ** 2: 1 / 6,
+                                                y ** 2: 1 / 3, x ** 3: 1 / 4,
+                                                x ** 2 * y: 1 / 6,
+                                                x ** 3 * y: 1 / 8,
+                                                x * y: 1 / 4, y ** 3: 1 / 4,
+                                                x ** 2: 1 / 3,
+                                                x * y ** 3: 1 / 8}
```

Comment:
You need to change `1 / 2` to `S(1) / 2`, the same with all the other ones like `1 / 9`, `1 / 5` and so on.